### PR TITLE
Update examples tracking the latest version of truffle

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

File: .github/workflows/build-examples.yml [hunk 1]
```diff
@@ -58,7 +58,15 @@ jobs:
         working-directory: examples/${{ matrix.example }}
         run: |
           if [ -f "scripts/deploy.js" ];then
-            npx truffle --network goerli exec scripts/deploy.js
+            # export RELEASE_VERSION=test
```
Comment (hellwolf) [inline .github/workflows/build-examples.yml:61]: the released npm has some bug related to this, will fix in next release

**Comment by hellwolf** [comment]
@pi0neerpat please have a look of the README changes.

Example project now tracking new truffle version again. Two issues pending to be fixed later:

- one test case was always failing, now the problem surfaced
- deploying to ganache using scripts from `node_module/@superfluid-finance/ethereum-contracts` is tricky, so the deployment tests are still done against goerli for now (which is not the most reliable way due to nonces of the shared build-agent account)

**Review by pi0neerpat — APPROVED** [review]
lgtm

**Comment by hellwolf** [comment]
> lgtm

thanks!

**Pull request merged**